### ISSUE TITLE
Setup issue

### ISSUE THREAD

**Issue by vinyvince** [taesung89/cyclegan#1]
Hi

Please pardon me once again, quick question in case you know about :

https://github.com/rozentill/pytorch-CycleGAN-and-pix2pix/commits/master

I followed all the install steps successfully on windows 10 x64 using the following steps
https://www.superdatascience.com/pytorch/
then ubuntu for windows with GitHub desktop
https://www.howtogeek.com/249966/how-to-install-and-use-the-linux-bash-shell-on-windows-10/

But im blocked at the latest steps written below for the correct installation , do download the database and start the training... :(
Your help would be be infinitely appreciated , once again i apologize to beg for your assistance...

Very Sincerely, take care and thanks in anycase

vincent*
bout.de.lune@gmail.com
------------------------------------------------

    Download a pix2pix dataset (e.g.facades):

bash ./datasets/download_pix2pix_dataset.sh facades

    Train a model:

#!./scripts/train_pix2pix.sh
python train.py --dataroot ./datasets/facades --name facades_pix2pix --model pix2pix --which_model_netG unet_256 --which_direction BtoA --lambda_A 100 --dataset_mode aligned --no_lsgan --norm batch --pool_size 0

    To view training results and loss plots, run python -m visdom.server and click the URL http://localhost:8097. To see more intermediate results, check out ./checkpoints/facades_pix2pix/web/index.html
    Test the model (bash ./scripts/test_pix2pix.sh):

#!./scripts/test_pix2pix.sh
python test.py --dataroot ./datasets/facades --name facades_pix2pix --model pix2pix --which_model_netG unet_256 

------------------------------------------------

-- 
________________________________________________________________

Vincent Thomas   (Avalailable January 2017)
Senior Lighting Lookdev / Env DigiMatte Artist /  Creative Designer VFX supervisor
 http://fr.linkedin.com/in/vincentthomas

**Comment by taesungp** [comment]
Hi Vincent, 

this repo is actually not the public repo. Please use https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix. 

I am not entirely sure which part you got stuck. What is the error message?

**Comment by vinyvince** [comment]
Hi Taesung

thanks so much for trying to help .
It's where i trying to download or trainthe model in windows. using for example the ubuntu linux prompt ..
i navigate manually to the appropriate place
type a following command like the one below, - i also tryied in su mode

"bash pretrained_models/download_pix2pix_model.sh facades_label2photo"

and i get immediate error 

root@Azure:/mnt/c/Users/vincent/Downloads/vision/pytorch-CycleGAN-and-pix2pix# bash pretrained_models/download_pix2pix_model.sh facades_label2photo
pretrained_models/download_pix2pix_model.sh: line 2: $'\r': command not found
Note: available models are edges2shoes, sat2map, and facades_label2photo
pretrained_models/download_pix2pix_model.sh: line 4: $'\r': command not found
]pecified [facades_label2photo
pretrained_models/download_pix2pix_model.sh: line 6: $'\r': command not found
mkdir: cannot create directory ‘./checkpoints/facades_label2photo\r_pretrained\r’: No such file or directory
pretrained_models/download_pix2pix_model.sh: line 10: $'\r': command not found
WARNING: timestamping does nothing in combination with -O. See the manual
for details.

: No such file or directoryhphoto
pretrained_models/download_pix2pix_model.sh: line 12: $'\r': command not found
pretrained_models/download_pix2pix_model.sh: line 13: $'\r': command not found

Any idea?
